calcom/cal.com
# Fix: about section editor

## What does this PR do?

This pr fixes the issue of Autofocusing of about section in profile page.

Fixes #6990

## Type of change


## How should this be tested?

- Go to Settings -> profile
- check the about section
